cmake/modules/SearchInstalledSoftware.cmake

```diff
@@ -511,11 +511,10 @@ if(asimage)
     else()
       set(_tiffinclude --with-tiff=no)
     endif()
-    if(cocoa)
-      set(_jpeginclude --without-x --with-builtin-jpeg)
-      set(_gifinclude  --with-builtin-ungif)
-      set(_pnginclude  --with-builtin-png)
-      set(_tiffinclude --with-tiff=no)
+    if(x11)
```

Comment:
It's true that `cocoa=ON` usually implies `x11=OFF`, but it's better to check the `x11` flag directly.